src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -263,10 +264,9 @@ public void moveToDefaultDirectory() {
 
     public boolean delete() {
         Optional<Path> file = linkedFile.findIn(databaseContext, Globals.prefs.getFileDirectoryPreferences());
-        if (file.isPresent()) {
-
-            ButtonType removeFromEntry = new ButtonType(Localization.lang("Remove from entry"));
+        ButtonType removeFromEntry = new ButtonType(Localization.lang("Remove from entry"));
```

Comment:
Can you please specify the button type [YES](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/ButtonBar.ButtonData.html#YES) so that the order of the buttons is correct.

Reply:
Moreover, I would suggest a more descriptive label like "(Only) unlink file" or "Remove file link".